run_tests.py

```diff
@@ -13,16 +13,12 @@
         TEMPLATE_LOADERS=(
             'django.template.loaders.app_directories.Loader',
         ),
```

Comment:
is this still needed? since `TEMPLATES` bellow has `APP_DIRS: True`

Reply:
No idea. But tests are passing so I'm less concerned. I'll know more once I really get in the swing of the upgrade.